app/controllers/api/v0/comments_controller.rb

```diff
@@ -13,8 +13,8 @@ class CommentsController < ApiController
       respond_to :json
 
       def index
-        @commentable = Article.find(params[:a_id])
-        @commentable_type = "Article"
+        article = Article.find(params[:a_id])
+        @comments = Comment.rooted_on(article, "Article").order(score: :desc)
```

Comment:
this was buried in the JSON view, by surfacing it we can instantly know how the comments are loaded and ordered in the API. It also paves the way for partial re-use